src/L10NSharp/XLiffUtils/XLiffFile.cs

```diff
@@ -0,0 +1,103 @@
+// ---------------------------------------------------------------------------------------------
+#region
+// Copyright (c) 2017 SIL International
+// This software is licensed under the MIT License (http://opensource.org/licenses/MIT)
+#endregion
+//
+// File: XLiffHeader.cs
+//
+// <remarks>
+// </remarks>
+// ---------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using System.Reflection;
+using System.Xml.Serialization;
+using System.Linq;
+
+namespace L10NSharp.XLiffUtils
+{
+	#region XLiffFile class
+	/// ----------------------------------------------------------------------------------------
+	/// <summary>
+	/// XLiff file
+	/// </summary>
+	/// ----------------------------------------------------------------------------------------
+	[XmlType("file", Namespace = "urn:oasis:names:tc:xliff:document:1.2")]
+	public class XLiffFile : XLiffBaseWithNotesAndProps
+	{
+		/// ------------------------------------------------------------------------------------
+		public XLiffFile()
+		{
+			SourceLang = "en";
+			ProductVersion = "0.0.0";
+			DataType = "csharp";
```

Comment:
The data in the xliff file is not c#. It could be text, it could be html, we don't know, but normally it's just text.  So I think 'plaintext' is what we want.